cmd/kubepug.go

```diff
@@ -84,6 +103,28 @@ func runPug(cmd *cobra.Command, args []string) error {
 	log.Debugf("Starting Kubepug with configs: %+v", config)
 	kubepug := lib.NewKubepug(config)
 
+	if config.Monitor {
+		done := make(chan os.Signal)
+		signal.Notify(done, os.Interrupt)
+		go func() {
+			http.Handle("/metrics", promhttp.Handler())
+			http.ListenAndServe("0.0.0.0:8888", nil)
+		}()
+
+		ticker := time.NewTicker(config.ScrapeInterval)
+		for {
```

Comment:
It seems that this leaves the program in an infinite loop, as an example:

```
time ./output/kubepug --monitor=false --k8s-version=v1.16.3 --scrape-interval=1m
[...]

real    3m14,432s
user    0m0,460s
sys     0m0,088s
```

I think it would be better to move the GetDeprecated to outside the cluster (as it only runs once anyway) and leave for the loop only the metrics scrapping, WDYT?